### ISSUE TITLE
[Issue] Missing packages: proton-tkg-git, vkd3d-proton-tkg-git wine-tkg-staging-fsync-git

### ISSUE THREAD

**Issue by eNTi** [chaotic-aur/packages#314]
Hi,

I am missing these packages. Why did you remove them if I may ask?

Kind regards

**Comment by PedroHLC** [comment]
> Why did you remove them if I may ask?
I'm migrating the building server, and I haven't finished migrating the service that was building those. So, to not keep people in outdated packages, I removed them :smile: 
You can still find them (outdated) in builds.garudalinux.org/repos/chaotic-aur/archive

They'll be rebuilt (up-to-date) after I finished migration...

**Comment by PedroHLC** [comment]
Hi, an update on this, the migration has ended, but I'm waiting on @Tk-Glitch to help me with an issue, so we can build his wines in both Singularity and NSpawn containers, safely, and without dirty hacks.

**Comment by mixalis1987** [comment]
Are the other wine-tkg builds coming back?
I only see wine-tkg-staging-fsync-git now.

Regards,

**Comment by PedroHLC** [comment]
🤔 it looks like _use_vkd3dlib no longers add -vkd3d to the package name
In other words we'll no longer provide -vkd3d variations...

the non-staging fails with:
```
1 out of 2 hunks FAILED -- saving rejects to file dlls/winevulkan/make_vulkan.rej
patching file dlls/winevulkan/vulkan_thunks.c
Hunk #1 succeeded at 3367 (offset 1 line).
Hunk #2 FAILED at 7111.
1 out of 2 hunks FAILED -- saving rejects to file dlls/winevulkan/vulkan_thunks.c.rej
patching file dlls/winevulkan/winevulkan.json
Hunk #1 FAILED at 2.
1 out of 1 hunk FAILED -- saving rejects to file dlls/winevulkan/winevulkan.json.rej
patching file include/wine/vulkan.h
Reversed (or previously applied) patch detected!  Skipping patch.
25 out of 25 hunks ignored -- saving rejects to file include/wine/vulkan.h.rej
```

LeagueOfLegends variation dies with:
```
../wine-mirror-git/configure: line 7604: syntax error near unexpected token `newline'
../wine-mirror-git/configure: line 7604: `    '''
```

**Comment by PedroHLC** [comment]
Sorry, this issue does not follow our new template.

Please, either:
1. open a new issue;
2. edit the first post to include [this template](https://github.com/chaotic-aur/packages/blob/main/.github/ISSUE_TEMPLATE.md) and then re-open this issue.
